verification/curator-service/ui/src/components/App.tsx

```diff
@@ -308,7 +308,6 @@ export default function App(): JSX.Element {
             })
             .catch((e) => {
                 setUser({ _id: '', name: '', email: '', roles: [] });
-                console.error(e);
```

Comment:
Why remove?

Reply:
This is going to print a console error for all logged out users, not super useful imo

![image](https://user-images.githubusercontent.com/1255432/92453282-5d06dd80-f1bf-11ea-8039-0caafa0a26f4.png)